src/flow/edmonds_karp.jl

```diff
@@ -1,18 +1,24 @@
-"""Computes the maximum flow between the source and target vertexes in a flow
-graph using [Edmond Karp's](https://en.wikipedia.org/wiki/Edmonds%E2%80%93Karp_algorithm)
+"""
+Computes the maximum flow between the source and target vertexes in a flow
+graph using [Edmonds Karp\'s](https://en.wikipedia.org/wiki/Edmondss%E2%80%93Karp_algorithm)
```

Comment:
This is `Edmonds-Karp` (two people; one named Jack Edmonds, the other Richard Karp).